docs/next/guides/getting-started/license-key.md

```diff
@@ -95,3 +93,18 @@ _The license key for Handsontable expired on `[expiration_date]`, and is not val
 ## Get a license key
 
 To get a commercial license key for your Handsontable copy, contact our [Sales Team](https://handsontable.com/get-a-quote).
+
+## Related articles
+
+#### Related guides
+
+- [Software license](@/guides/technical-specification/software-license.md)
+
+#### Related blog articles
+
+- [Handsontable drops open source for a non-commercial license](https://handsontable.com/blog/articles/2019/3/handsontable-drops-open-source-for-a-non-commercial-license)
```

Comment:
Please remove. Irrelevant after 3 years we did it.

Reply:
Done: [fd92fc9](https://github.com/handsontable/handsontable/pull/9374/commits/fd92fc93482cf3e75493cea4074e5bea07daa9b5)